usr/share/rear/conf/default.conf

```diff
@@ -949,3 +949,5 @@ export TMPDIR    # the export is required so that mktemp can pickup the variable
 # Specify if rear is managed from DRLM (y/n) [ default (n) ].
 DRLM_MANAGED=n
 
+# Specify if rear should try to backup capabilities (y/n) default (n).
+BACKUP_CAP=n
```

Comment:
Please define it as `NETFS_RESTORE_CAPABILITIES=` in the `default.conf` file. To check if a variable is true or not try to use the function `is_true` (defined in lib/global-functions.sh)